config/locales/en.yml

```diff
@@ -2786,6 +2786,32 @@ en:
           primary_disabled_question: Are you disabled according to the New Jersey definition?
           spouse_disabled_question: Is your spouse disabled according to the New Jersey definition?
           title: Disabled Exemption
+      nj_eitc_qualifying_child:
+        edit:
+          helper_contents_html: |-
+            <p>You are the qualifying child of another taxpayer for the Earned Income Tax Credit (EITC) in 2024 if <strong>all</strong> of these are true.</p>
+            <p>1. You are that person's:</p>
```

Comment:
[pebble] this can be made into semantic html with an `ol` instead of p tags

Reply:
done